test/examples-golden/positive/llvm/Instructions/Instructions.llvm

```diff
@@ -4,46 +4,53 @@
  
 
 
-define external ccc  i8 @lam_0(i8  %arg_0)    {
-  ret i8 99 
+define external fastcc  i8 @"\22lambda\226"(i8  %"5")    {
+"\22lambda\226":
```

Comment:
Hmm what is up here? Problem with variable name printing?

Reply:
Seems the unique name generator I have generates
`"\"lambda\"1"`

we can create an issue after this PR to clean it up, as it just ugly rather than stopping operation